### ISSUE TITLE
gtk3 UI corruption

### ISSUE THREAD

**Issue by SpaceRaven** [Alexey-Yakovenko/deadbeef#1267]
When i add cover field to deadbeef-0.6.2 whole UI become currupted. I tested it on various distros, gentoo, fedora within various DE(gnome/kde). Look into screeshot

![2015-09-12 11-36-18](https://cloud.githubusercontent.com/assets/14246735/9830797/8526520a-5944-11e5-8b31-33d5d1fb7f8f.png)

**Comment by Alexey-Yakovenko** [comment]
This is fixed in the following commit:

```
commit c7b1bfa2716c2dce9ea5d099e049e80e6db9890d
Author: Christian Boxdörfer <christian.boxdoerfer@posteo.de>
Date:   Tue Sep 9 09:23:33 2014 +0200

    gtkui: fix visual artifacts in gtk3 version (album art showed up in status and toolbar etc.)
```

Please wait until the release 0.7, or build from git master branch, or stick to using a different GTK3 theme, or use GTK2.

**Comment by Alexey-Yakovenko** [comment]
btw, the fix has been backported to the official 0.6.2 builds, so you can use one of those too.
